### ISSUE TITLE
Black screen windows 10  HELP

### ISSUE THREAD

**Issue by Stefano-82** [libretro/RetroArch#7066]
Hello everyone,

i just got retroarch on 2 different systems, one running windows 7 with  an intel i5 5460 with no discrete gpu, this setup runs with gl video drivers smoothly, graphics look really really polished and i almost see no pixels around text and 3d elements.

The other setup on windows 10 + i5 4440 + R9270x with the same setup with GL gets a black screen, changing it into d3d11 the game runs.

The main issue i notice is that the graphics with the same exact setup look a lot more polished with the gl running while the latter system is more beefy, 

Is there a chance GL might impact that much compared to d3d11 and if so can you help with fixing this issue?

Thanks in advance

**Comment by Stefano-82** [comment]
I just found what the problem was, after several hours, apparently when you choose GL driver and set(in my case) x4 internal resolution + xbr and pgxp enabled with dithering x1, the game runs with a good filter.

In the other case if you pick d3d11, don't ask me the reason but with the same settings you get almost the RAW graphic.
I solved it working with the shaders "LANG", and using X6BRZ i get the same results as with GL drivers.

So i guess it can be closed, it was a bit tricky but luckily i had time to spend :)


cheers
